MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,643 @@
+
+Here, `FirstName` data field will be changed from the default position.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.Padding = new thickness(10,0,10,0);
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing label name
+
+By default property name should be displayed as label. You can change the label text by using display attribute or using the `GenerateDataFormItem` event.
+
+### Using attribute
+
+You can change the label text by setting the [Name]`Link` in display attribute.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    
+    [Display(Name = "FirstName")]
+    public string Name { get; set; }
+    
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+You can change the label text by using the [LabelText] `Link` property in [DataFormItem] `Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LabelText = "FirstName";
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Add or remove Label text for editor
+
+By default the label should be generated by property name. You can remove label text by using the DataFormDisplayAttribute or using the `GenerateDataFormItem` event.
+
+### Using attribute
+
+You can remove the label text for editor by setting [ShowLabel] as false in dataformdisplayOption attribute.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string MiddleName { get; set; }
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Using events
+
+You can remove the label text for editor by using the [ShowLabel] `Link` property in [DataFormItem] `Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ShowLable = false;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing layout settings
+
+You can change the label width, label position and editor width. This is possible by using the `GenerateDataFormItem` event. In the event, change the layout by using the [DefaultLayoutOption] property in the `DataFormItem`.
```

Comment:
DefaultLayoutOption => DefaultLayoutSettings

Reply:
DefaultLayoutSettings added instead of DefaultLayoutOption